UnityProject/Assets/Scripts/Managers/PlayerInfo.cs

```diff
@@ -142,6 +144,11 @@ private void TryChangeName(string playerName)
 		name = uniqueName;
 	}
 
+	public void SetMind(Mind InMind)
```

Comment:
Why not make the setter public?

Reply:
more implicit way of setting the mind, so something is a bit more big than just setting A variable